package.json

```diff
@@ -83,7 +82,7 @@
     "pinst": "^2.1.6",
     "prettier": "^2.2.1",
     "semantic-release": "^17.3.0",
-    "stylelint": "^13.8.0",
+    "stylelint": "^14.0.1",
```

Comment:
Styleling `v14` comes with secondary options:

- [ ] [`disableFix`](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-14.md#disablefix-secondary-option) or [here](https://github.com/stylelint/stylelint/blob/main/docs/user-guide/configure.md#disablefix)

- [ ] [`message`](https://github.com/stylelint/stylelint/blob/main/docs/user-guide/configure.md#message)